src/themes.css

```diff
@@ -78,24 +78,124 @@
 }
 
 [data-theme="vs-code"] {
-    --p-text : rgb(220, 220, 170);
-    --canvas-bg : rgb(31, 31, 31);
-    --canvas-items : rgb(156, 220, 254);
-    --good-placement : rgb(115, 201, 145);
-    --bad-placement : rgb(244, 71, 71);
-    --header-border: 1px solid rgb(204, 204, 204);
-    --header-bg : rgb(43, 43, 43);
-    --toolbar-border: 1px solid rgb(204, 204, 204);
-    --toolbar-bg: rgb(43, 43, 43);
-    --subbar-border: 1px solid rgb(204, 204, 204);
-    --subbar-bg: rgb(43, 43, 43);
-    --sidebar-border: 1px solid rgb(204, 204, 204);
-    --sidebar-bg: rgb(43, 43, 43);
-    --mode-btn-border: 1px solid rgb(204, 204, 204);
-    --mode-btn-bg: rgb(43, 43, 43);
-    --mode-btn-icon-color : rgb(134, 134, 134);
+    --p-text : #DCDCAA;
+    --canvas-bg : #1F1F1F;
+    --canvas-items : #9CDCFE;
+    --good-placement : #73C991;
+    --bad-placement : #F44747;
+    --header-border: 1px solid #CCCCCC;
+    --header-bg : #2B2B2B;
+    --toolbar-border: 1px solid #CCCCCC;
+    --toolbar-bg: #2B2B2B;
+    --subbar-border: 1px solid #CCCCCC;
+    --subbar-bg: #2B2B2B;
+    --sidebar-border: 1px solid #CCCCCC;
+    --sidebar-bg: #2B2B2B;
+    --mode-btn-border: 1px solid #CCCCCC;
+    --mode-btn-bg: #2B2B2B;
+    --mode-btn-icon-color : #868686;
     --mode-btn-radius: 10%;
     --mode-btn-hover-bg: none;
     --mode-btn-shadow: none;
     --mode-btn-press-shadow: none; 
 }
+
+/* Ryan Theme */
+[data-theme="Peirced-from-Within"] {
+    --dark-highlights : #B11D20;
+    --p-text : #F5D51D;
+    --canvas-bg : #07140E;
+    --canvas-items : #F4F3CD;
+    --good-placement : #46FF00;
+    --bad-placement : #C60003;
+    --header-border: 1px solid #0A1E0A;
+    --header-bg : #151715;
+    --toolbar-border: 1px solid #02281C;
+    --toolbar-bg: #150F12;
+    --subbar-border: 1px solid #05281C;
+    --subbar-bg: #201412;
+    --sidebar-border: 1px solid #05281C;
+    --sidebar-bg: #05141D;
+    --mode-btn-border: 1px solid #05281C;
+    --mode-btn-bg: r#2B2B2B;
+    --mode-btn-icon-color : #868686;
+    --mode-btn-radius: 0%;
+    --mode-btn-hover-bg: none;
+    --mode-btn-shadow: none;
+    --mode-btn-press-shadow: none; 
+}
```

Comment:
For consistency we should decide if we're doing capital letters for Hex values or lower case letters.